apps/opencs/view/render/terrainstorage.hpp

```diff
@@ -15,12 +16,25 @@ namespace CSVRender
     {
     public:
         TerrainStorage(const CSMWorld::Data& data);
+        float mAlteredHeight[ESM::Land::LAND_SIZE * ESM::Land::LAND_SIZE + ESM::Land::LAND_SIZE];
```

Comment:
It is a general recommendation for C++ to use `std::array` instead of raw arrays https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#es27-use-stdarray-or-stack_array-for-arrays-on-the-stack . Please consider to use it or explain why it is not applicable here.

Reply:
I think it's applicable. I just hadn't heard of std::array before, nor the advantages. Thanks for the link!